HospitalGame_Business/src/business/BusinessFacade.java

```diff
@@ -347,39 +359,65 @@ public String interact(IPlayer player, INPC npc) {
     }
 
     /**
-     * 
+     *
      * @return the score if the game if won else it returns 0.
      */
     @Override
     public int getScore() {
         return score;
     }
 
+    /**
+     * Checks if eligible for a highscore
+     *
+     * @return true is its the score is eligible for a highscore
+     */
     @Override
     public boolean eligibleForHighScore() {
         return highScore.eligibleForHighscore(score);
     }
 
+    /**
+     * Adds a highscore
+     *
+     * @param name of the player
+     * @return true if the highscore is to be added
+     */
     @Override
     public boolean addHighScore(String name) {
         return highScore.addHighScore(name, score);
     }
 
+    /**
+     * if the highscore has changed, then save this highscore
+     */
     @Override
     public void closing() {
```

Comment:
Not correct.
This should be something like:
This function should be called before the game is closed